src/include/homestore/btree/detail/btree_node.hpp

```diff
@@ -442,8 +472,8 @@ class BtreeNode : public sisl::ObjLifeCounter< BtreeNode > {
     void set_link_version(uint64_t version) { get_persistent_header()->link_version = version; }
     void inc_link_version() { ++(get_persistent_header()->link_version); }
 
-    void set_valid_node(bool valid) { get_persistent_header()->valid_node = (valid ? 1 : 0); }
-    bool is_valid_node() const { return get_persistent_header_const()->valid_node; }
+    void set_node_deleted() { get_persistent_header()->node_deleted = 0x1; }
```

Comment:
valid_node name is ambigious since we use it for other reasons for valid check, so made it explicit that node is deleted.